spec/requests/portfolios_rbac_spec.rb

```diff
@@ -45,7 +60,10 @@
     before do
       allow(Insights::API::Common::RBAC::Roles).to receive(:assigned_role?).with(catalog_admin_role).and_return(false)
     end
+
     it 'returns status code 200' do
+      permission = 'catalog:portfolios:read'
+      create(:access_control_entry, :group_uuid => group1.uuid, :permission => permission, :aceable => portfolio1)
```

Comment:
Could probably move this and Line 44 out to a let since they're the same line sans permission.